app.rb

```diff
@@ -0,0 +1,7 @@
+require 'sinatra/base'
+
+class App < Sinatra::Base
+  get '/' do
+    'Hello'
```

Comment:
For authentication health checks we would want to do an actual authentication of the HEALTH user to make sure that the entire backend stack is up. Something to consider is that this would hit the cache instead of the DB most of the time. 
On that point, thinking a bit forward, we would want to make sure that the DB connections are lazy.

Reply:
Given that this isn't currently connecting to the database, I'm happy for this to be a placeholder now and change it as part of connecting to the DB if that works for you

Reply:
Sure thing, let's get it movin.